### ISSUE TITLE
(IR) re-implement update-catalog robot (blocked by PC #109, #219)

### ISSUE THREAD

**Issue by ndushay** [sul-dlss/sdr-preservation-core#85]
this is likely moot as this code base will be replaced by sul-dlss/preservation_robots

from sul-dlss/preservation2017/issues/45 (7)

"This step writes to the current Archive Catalog and will have to be altered to write to the PCC Inventory and Provenance."

We want this to call "create" or "update_version" via ReST in preservation_catalog.

This probably involves:

- [ ] update `lib/sdr_ingest/update_catalog.rb` and its spec
- [ ] update `config/robots.yml`
- [ ] update `config/workflows/sdrIngestWF/sdrIngestWF.xml`
- [ ] update `config/workflows/sdrIngestWF/workflowDefinition.xml` 
- [ ] update README

also? 
- [ ] update `config/environments/robots_development.yml`
- [ ] update `config/workflows/sdrIngestWF/process-config.yaml` ?
- [ ] update scripts in bin/

**Comment by ndushay** [comment]
superseded by sul-dlss/preservation_robots#7
